provisioning/travis_ci_and_cd/build_and_push_to_ecr.sh

```diff
@@ -2,11 +2,10 @@
 # Push only if it's not a pull request
 if [ -z "$TRAVIS_PULL_REQUEST" ] || [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
   # Push only if we're testing a deployable branch
-  if [ "$TRAVIS_BRANCH" == "qa" ] || [ "$TRAVIS_BRANCH" == "production" ]; then
-
-        export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID_PRODUCTION
-        export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY_PRODUCTION
-        DOCKER_REPO_URL=$REMOTE_IMAGE_URL
+  if [ "$TRAVIS_BRANCH" == "develop" ]; then
+    export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID_DEVELOPMENT
```

Comment:
Just a note: these keys are definitely out of date at this point, and when we start coordinating on deploying with Docker, this project will need to have the rotation done.